api/swagger.yaml

```diff
@@ -9865,6 +9880,7 @@ paths:
           schema:
             type: "object"
             x-go-name: DistributionInspect
+            title: "DistributionInspect"
```

Comment:
`DistributionInspectResponse`

Reply:
Here I used the same as x-go-name, as I assume this one is here for some reason. Maybe this doesn't make sense ?

Reply:
Ok, sounds fair, we can keep it as is